README.md

```diff
@@ -1 +1,21 @@
-# dont-panic-api
\ No newline at end of file
+# dont-panic-api [WIP]
+Work in progress - the current version doesn't have the basic features yet and shouldn't be used
+
+This is an api written to support your studies in competitive programming.
+It intends to be a way to track the problems you've solve, the subjects you've learned and suggests new subjects to learn and when its time to review any of them by suggesting problems with these subjects.
```

Comment:
"It intends" não parece ser a expressão certa, dado que o programa em si não tem intenção por si mesmo. É melhor algo tipo: O objetivo desse projeto é ...

Reply:
"[...] the subjects you've learned and suggests new [...]"
esse 'suggests' está continuando o 'to be a way to', então deveria ser 'suggest'

Reply:
Muito confuso isso tudo. É uma lista de coisas que deixa tudo confuso, a tradução direta tá: "Isso pretende ser um jeito de acompanhar os problems que você resolve, os assuntos que aprendeu and sugerir novos assuntos pra aprender e quanto é o momento de revisar algum deles sugerindo problemas desses assuntos". Essa lista começa bem direta, aí começa a falar de sugerir assuntos, depois começa a colocar o tempo pra sugerir problemas dos assuntos.

Algo mais direto ficaria bem mais fácil de entender: "Acompanha os problemas resolvidos e assuntos estudados, e sugere novos assuntos e problemas." Se for pra explicar melhor como funciona a parte do tempo, explica em uma parte separada